src/sources/soundsourcecoreaudio.cpp

```diff
@@ -4,6 +4,20 @@
 
 namespace Mixxx {
 
+namespace {
+
+// NOTE(rryan): For every seek we jump back kStabilizationFrames frames from the
+// seek position and read forward to allow the decoder to stabilize. The
+// cover-test.mp3 file needs this otherwise SoundSourceProxyTest.seekForward
+// fails. I can't find any good documentation on this on the web. Oddly, the
+// "prime" information -- which AIUI is supposed to tell us this information --
+// is zero for this file.
+static const int kStabilizationFrames = 2500;
+static CSAMPLE kStabilizationScratchBuffer[kStabilizationFrames *
```

Comment:
I'd rather not have to build libmad on mac. We will always have to use SSCoreAudio for M4A/AAC support so this doesn't actually save us any code.